src/controllers/user.controller.js

```diff
@@ -0,0 +1,93 @@
+const {
+  getUsers,
+  addUser,
+  getUser,
+  deleteUser,
+  updateUser,
+} = require('../services/users.service');
+
+const getAll = (req, res) => {
+  res.statusCode = 200;
+
+  const users = getUsers();
+
+  res.send(users);
+};
+
+const add = (req, res) => {
+  const { name } = req.body;
+
+  if (!name) {
+    res.sendStatus(400);
+  }
```

Comment:
Consider adding a `return` statement after `res.sendStatus(400);` to prevent further execution of the function after sending a response.